src/test/java/api/ApiLaunchesTests.java

```diff
@@ -0,0 +1,56 @@
+package api;
+
+import org.junit.jupiter.api.Tag;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import jdk.jfr.Description;
+
+import java.io.File;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import io.restassured.response.Response;
+
+import static api.LaunchesApiClient.deleteLaunch;
+import static api.LaunchesApiClient.getLaunchID;
+import static api.LaunchesApiClient.updateLaunch;
+import static api.endpoints.LaunchesApi.LAUNCHES;
+
+public class ApiLaunchesTests extends ApiBaseTest {
```

Comment:
I would recommend to move this class to the api_tests package together with the other test classes for the API module.

Reply:
Fixed